src/languages/javascript.coffee

```diff
@@ -108,5 +108,9 @@ module.exports = {
       default: "System Default"
       enum: ["CRLF","LF","System Default"]
       description: "Override EOL from line-ending-selector"
+    bracket_spacing:
```

Comment:
How about `object_curly_spacing` to align with Unibeautify and ESLint? See https://unibeautify.com/docs/option-object-curly-spacing.html  

Or `object_bracket_spacing`?